src/app/data-planes/components/RuleEntryList.vue

```diff
@@ -19,87 +19,74 @@
         <div class="policy-list">
           <KTable
             class="policy-type-table"
-            :fetcher="() => ({ data: ruleEntry.connections, total: ruleEntry.connections.length })"
-            :headers="tableHeaders"
+            :class="{
+              'policy-type-table--with-matchers': props.showMatchers,
+            }"
+            :fetcher="() => ({ data: ruleEntry.rules, total: ruleEntry.rules.length })"
+            :headers="[
+              ...(props.showMatchers ? [{ label: 'Matchers', key: 'matchers' }] : []),
+              { label: 'Origin policies', key: 'origins' },
+              { label: 'Conf', key: 'config' },
+            ]"
```

Comment:
Not looking too much into it myself I wondered why this couldn't use AppCollection? I guess there might be more reasons apart from the fact that the code being changed here came before we had AppCollection?